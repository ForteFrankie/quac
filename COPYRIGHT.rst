Copyright
=========

QUAC is copyright © 2012–2019 Triad National Security, LLC and others.

This program was produced under U.S. Government contract 89233218CNA000001 for
Los Alamos National Laboratory (LANL), which is operated by Triad National
Security, LLC for the U.S. Department of Energy/National Nuclear Security
Administration. The Government is granted for itself and others acting on its
behalf a nonexclusive, paid-up, irrevocable worldwide license in this material
to reproduce, prepare derivative works, distribute copies to the public,
perform publicly and display publicly, and to permit others to do so.

This is open source software. Except as specified in the above, you may
redistribute it and/or modify it only under the terms of the Apache License,
Version 2.0. If software is modified to produce derivative works, such
modified software should be clearly marked, so as not to confuse it with the
version available from LANL. Full text of the License is available in the file
LICENSE or at http://www.apache.org/licenses/LICENSE-2.0.

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
License for the specific language governing permissions and limitations under
the License.

This project contains code from Cyclopath, which is copyright © 2006–2012
Regents of the University of Minnesota and open source under the Apache
License. Grep for "Cyclopath" to find the relevant files.

*This software has been approved for open source release, LA-CC 12-038 and
LA-CC 15-044.*
