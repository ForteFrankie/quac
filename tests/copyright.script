#!/bin/bash

# Print the name of each file that does not contain the copyright notice
# (below); the test then compares this against a list of exceptions (the
# .stdout file).

C='Copyright (c) 2012-2013 Los Alamos National Security, LLC, and others'


. ./environment.sh

cd ..

for filename in $(git ls-files); do
    if (echo $filename | egrep -qv '(\.stdout$|\.stderr$)'); then
        fgrep -iL "$C" $filename || true
    fi
done
