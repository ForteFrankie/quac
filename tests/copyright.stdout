.gitignore
COPYRIGHT.rst
LICENSE
lib/disco/README
lib/geo/__init__.py
lib/qr/__init__.py
lib/unicodedata2.py
sphinx/Makefile
sphinx/conf.py
tests/README
twtests/README
twtests/tweets/ids.txt
